public/app/features/playlist/PlaylistEditPage.tsx

```diff
@@ -25,16 +26,22 @@ export const PlaylistEditPage = ({ match }: Props) => {
   };
 
   const pageNav: NavModelItem = {
-    text: 'Edit playlist',
-    subTitle:
-      'A playlist rotates through a pre-selected list of dashboards. A playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.',
+    text: t('playlist-edit.title', 'Edit playlist'),
+    subTitle: t(
+      'playlist-edit.subTitle',
+      'A playlist rotates through a pre-selected list of dashboards. A playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.'
+    ),
   };
 
   return (
     <Page navId="dashboards/playlists" pageNav={pageNav}>
       <Page.Contents isLoading={playlist.loading}>
-        {playlist.error && <div>Error loading playlist: {JSON.stringify(playlist.error)}</div>}
-
+        {playlist.error && (
+          <div>
+            <Trans i18nKey="playlist-edit.errorPrefix">Error loading playlist:</Trans>
```

Comment:
```suggestion
            <Trans i18nKey="playlist-edit.error-prefix">Error loading playlist:</Trans>
```
(i realise `errorPrefix` was originally my suggestion, apologies for not suggesting in kebab-case)